content/source/docs/enterprise/getting-started/policies.html.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: "enterprise2"
+page_title: "Managing Sentinel Policies - Getting Started - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-started-policies"
+---
+
+# Creating and Managing Terraform Policies
+
+**Prerequisites:** Before starting this guide, make sure you've [sucessfully completed a run](./runs.html).
+
+## About Sentinel Policies
+
+_Policies_ in TFE are composed of [Sentinel policies](../sentinel/index.html) with some extra features. [Sentinel](https://www.hashicorp.com/sentinel) is an embedded policy-as-code framework integrated with the HashiCorp Enterprise products. It enables fine-grained, logic-based policy decisions, and can be extended to use information from external sources. Within TFE, you can use Sentinel to apply checks to your runs.
+
+A policy consists of:
+
+- The Sentinel policy code
+- An enforcement mode that changes how a policy affects the run lifecycle
+
+Currently, policies in an organization apply to **all** workspaces. You'll want to consider that when writing your policies so they won't fail unrelated workspace runs or configurations. In the future, TFE will provide a way to manage policies so you can apply them per-workspace.
+
+## Creating a Policy
+
+First, make sure you're viewing the organization settings. If you're still on a run page (or any other page), click the "Settings" button in the top navigation bar.
+
+![navigate](./images/policy-navigate.png)
+
+The policy list shows all of the policies you have access to; if you haven't created any, it's empty.
+
+To create your first policy, click the "Create new policy" button in the upper right.
+
+![new button](./images/policy-new-button.png)
+
+On the "Create a new Policy" page, you need to enter at least two items: a policy name, and the policy code. When you've finished, click the "Create Policy" button.
+
+![fields on new policy page](./images/policy-new-fields.png)
+
+### Policy Name
+
+A policy name should tell your colleagues what the policy is for. Most policies are _a particular environment_ of _a particular Terraform configuration,_ so the name should include both the name of the configuration and the name of the environment.
```

Comment:
> Most policies are a particular environment of a particular Terraform configuration

* Should probably read "Most policies are _for_ a particular environment of a particular Terraform configuration".

On a more fundamental level, I would be careful in assuming that even _most_ policies are going to be specific for environments or workspaces - while there will be a number that do for sure, I would expect a well-laid out governance structure to treat these as exceptions rather than the rule, as governance and policy should apply to the organization globally (based on the principle that a large amount of exceptions dilutes and hinders good governance). I think we should be encouraging policy authors to structure their policies and governance so that they can catch a good number of their cases without making exceptions.

Some examples of good organization-wide governance material:

* The classic example of ensuring that all AWS assets have tags assigned to them (as we have on the Sentinel website). This could even be further drilled into to ensure that the tag matches a certain pattern (I used to encourage a pattern of `ORG/PROJECT`).
* Ensuring all deployed network assets contain a general set of network ACLs.
* Ensuring all of your `aws_instance` data sources [are looking up AMIs on a specific owner](https://github.com/terraform-providers/terraform-provider-aws/pull/5576).

I think we should probably not necessarily imply a best practice here, and save this material for later. Maybe we could just edit this to:

> Policy Name
>
> A policy name should tell your colleagues what the policy is for. Examples could be "Require tags on all instances" or "Enforce network ACLs". If a policy is for a particular workspace, or environment within a workspace, include the name of the workspace and/or environment.

Reply:
Oh dang. That was actually left over from copy/pasting the Workspace text. Will fix. Thanks!